Sources/CLI/Network/NetworkList.swift

```diff
@@ -22,31 +22,32 @@ import Foundation
 import SwiftProtobuf
 
 extension Application {
-    struct NetworkList: AsyncParsableCommand {
-        static let configuration = CommandConfiguration(
+    public struct NetworkList: AsyncParsableCommand {
+        public init() {}
+        public static let configuration = CommandConfiguration(
             commandName: "list",
             abstract: "List networks",
             aliases: ["ls"])
 
-        @OptionGroup
-        var global: Flags.Global
-
         @Flag(name: .shortAndLong, help: "Only output the network name")
         var quiet = false
 
         @Option(name: .long, help: "Format of the output")
         var format: ListFormat = .table
 
-        func run() async throws {
+        @OptionGroup
+        var global: Flags.Global
+
+        public func run() async throws {
             let networks = try await ClientNetwork.list()
             try printNetworks(networks: networks, format: format)
         }
 
-        private func createHeader() -> [[String]] {
+        func createHeader() -> [[String]] {
```

Comment:
Same here...unless you really need these today for your use case, I'm inclined to keep these private and focus on coming up with a more reusable solution for tabular output.